ports/mire/port.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": 3,
+    "name": "mire.zip",
+    "items": [
+        "Mire.sh",
+        "mire/"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Mire",
+        "porter": [
+            "MadShmupper"
+        ],
+        "desc": "Mire is like a very small metroidvania, but nothing is gated. You can visit any area, in any order. There are 'upgrades' that you might find useful but they are not necessary, and you can just pick up the ones you want. It's more about exploration, of seeing something at the corner of the screen and figuring out how to get there.",
+        "desc_md": null,
+        "inst": "Ready to Run",
+        "inst_md": null,
+        "genres": [
+            "adventure",
+            "platformer"
+        ],
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
```

Comment:
You added a port.json from an installed copy. None of the bottom fields should be in it =)